### ISSUE TITLE
credits, release notes, readme, contributors tab, citations

### ISSUE THREAD

**Issue by illume** [pygame/pygame#2297]
How do we credit people better? 

Currently there are credits in:
- there's some pre-github people credited in the README.rst file
- contributors section of github lists some github contributors (but not all)
- some notes in the old WHATSNEW file thank people
- github release notes.

Some notes from a discussion about it earlier:

> It would have been nice if the new contributors had been in the readme
> I feel that contributors list can really make the README too long

> One problem with github contributors list is it doesn't really work for CVS/SVN times when email patches were much more common. I have a feeling it's not possible to add authors to previous commits without fucking up the git repo.

> I mean, how do you rank the contributions? Do you rank by age/tenior, amount of commits, impact of work, alphabetical, random? Do you spend time thanking people individually. Do we spend time making a credits video? Put the credits in a game? How about academic credits? Do people who contributed 15 years ago get to keep having their name put up as the single person responsible for it all? Do you only credit people in the release notes? Do you rip away credits from people just because they stopped contributing some years ago, even though the project wouldn't exist today without them?

> The idea about writing references for people, so they can put in job applications/linkedin is one that seems impactful. Another one for academics is having something people can add citations for. Since getting citations affects everything from grants to jobs in some places. Game credits are a thing if you want to work in some studios. Having a personal thank you can mean a lot to some.

Related PR about adding a docs/contributors.txt: https://github.com/pygame/pygame/pull/2294

Some releases didn't have detailed release notes, some are in WHATSNEW file and some are in github releases.

**Comment by siggisv** [comment]
One suggestion is that we could do the following:

- In the credits section of `README.rst`:
   1. Start with a short text about Pete Shinners being the original author in 2000 and having been the [main contributor / maintainer / manager / whatever would be the most appropriate title] for the first X years before moving on to other things.
   2. Then a short text stating who is the current [maintainer / manager / project leader / whatever would be the most appropriate title for the role @illume has been filling] and for how long that has been. If there are any that came before him, mention those and when.
   3. Talk about there being a long list of people who helped and contributed through the years, without whom this project wouldn't exist today (and a link to a file/web-page containing lists of those people).
   4. A thank you to those that helped/contributed to the most resent stable release, listed in whichever way feels appropriate (could be a copy/paste from the most resent release note, or alphabetical list).
   5. And finally some thank you to whoever would be appropriate to add (hosting? making tutorials? etc).
- In either a separate file (within `docs`?) or just on the web site:
   Starting with the newest stable release, list those that helped/contributed to each stable release, until we get to the pre-github releases, where we can just state "Release x.y.z and before:" and then insert the credits as they were then in the `README.rst` file.

The hardest work would be to go through the releases notes, WHATSNEW files, etc. to get the info for the file/page containing the list of previous credits. But once that is done, then at each new stable release, the credits section of the release notes could just be copy/pasted into section iv of README.rst`  and also added to the front of the list of historical credits.

**Comment by illume** [comment]
Yeah, I think back filling the previous github releases (from WHATSNEW and emails) could be a good thing start on, since that would be good to do anyway.

Also, I agree putting them into files like docs/releases/1.9.6.rst would be good. I don't think that would add too much extra time when adding releases. The releases have to be in mark down format on github, but .rst format for our docs/ ... probably a script could automate this.

The website about/ page could probably link to some credits, probably previous releases. Note, that now I've put pygame as a 'project' on the website with all those credits. When the `docs/releases/` notes are there, that means a lot more on the website.

I'm not sure about the README, but removing the credits section there and linking to the /about page (or credits pages whereever they are) on the website is probably a good idea. One nice thing about the existing credits section of the README is that it mentions some things about things people worked on, rather than just a name.
